go/pkg/bertyprotocol/store_metadata.go

```diff
@@ -286,6 +286,31 @@ func metadataStoreAddEvent(ctx context.Context, m *metadataStore, g *bertytypes.
 	return op, nil
 }
 
+// ContactReady informs if secrets are sent to the member, so if it can read messages send by us
+func (m *metadataStore) ContactReady(ctx context.Context, memberPK crypto.PubKey) (bool, error) {
```

Comment:
```suggestion
func (m *metadataStore) ContactReady(ctx context.Context, memberPK crypto.PubKey) error {
```

bool is useless here, it's always `err != nil`

Reply:
Resolved because this code will be removed, see #2312 (comment)